docs/source/Mux.rst

```diff
@@ -19,63 +19,63 @@ in reality it follows the way people are used to define dependencies,
 therefor it's very simple to use and clear even in complex cases.
 
 The best explanation comes usually from examples, so feel free to scroll down
-to `yaml_to_mux plugin`_ section, which uses the default mux plugin to feed
-the Mux.
+to `yaml_to_mux plugin`_ section, which uses the default variants plugin to feed
+the Variants.
 
 
-Mux internals
--------------
+Varianter internals
+-------------------
 
-The ``Mux`` is a core part of avocado and one can see it as a ``multiplexed``
+The ``Varianter`` is a core part of avocado and one can see it as a ``multiplexed``
 database, which contains key/value pairs associated to given paths and
 as we are talking about a tree of those, we call the paths ``Nodes``.
 
-Mux allows iterating through all possible combinations which are stored in
-the database, which is called ``multiplexation``. Mux yields ``variants``,
+Varianter allows iterating through all possible combinations which are stored in
+the database, which is called ``multiplexation``. Varianter yields ``variants``,
```

Comment:
Well currently only `multiplexer` is available in `varianter`. We define i as `multiplexed` database. Then we say it allows iterating throught combinations called `multiplexation`, which produces `variants`.

I think it's actually consistent with the current features and when we allow different varianter plugins, we have to rewrite this from scratch as the difference is too big. Do you really think it's worth trying to fix this version (which contains several other issues already)?